packages/web/app/src/stories/tabs.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from 'react';
+import { Tabs, TabsList, TabsListMenu, TabsTrigger, TabsTriggerMenu } from '@/components/ui/tabs';
+import { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof Tabs> = {
+  title: 'Components/Tabs',
+  component: Tabs,
+};
+
+export default meta;
+type Story = StoryObj<typeof Tabs>;
+
+export const Default: Story = {
+  render: () => {
+    const Page = {
+      Overview: 'overview',
+      Members: 'members',
+      Policy: 'policy',
+      Settings: 'settings',
+      Support: 'support',
+      Subscription: 'subscription',
+    };
+    const [page, setPage] = useState(Page.Overview);
+
+    return (
+      <Tabs value={page} className="min-w-[600px]">
+        <TabsList>
+          <TabsTrigger value={Page.Overview} asChild>
+            <a onClick={() => setPage(Page.Members)}>Overview</a>
+          </TabsTrigger>
+          <TabsTrigger value={Page.Members} asChild>
+            <a onClick={() => setPage(Page.Members)}>Members</a>
+          </TabsTrigger>
+          <TabsTrigger value={Page.Policy} asChild>
+            <a onClick={() => setPage(Page.Policy)}>Policy</a>
+          </TabsTrigger>
+          <TabsTrigger value={Page.Settings} asChild>
+            <a onClick={() => setPage(Page.Settings)}>Settings</a>
+          </TabsTrigger>
+          <TabsTrigger value={Page.Support} asChild>
+            <a onClick={() => setPage(Page.Support)}>Support</a>
+          </TabsTrigger>
+          <TabsTrigger value={Page.Subscription} asChild>
+            <a onClick={() => setPage(Page.Subscription)}>Subscription</a>
```

Comment:
```suggestion
            <button onClick={() => setPage(Page.Members)}>Overview</button>
          </TabsTrigger>
          <TabsTrigger value={Page.Members} asChild>
            <button onClick={() => setPage(Page.Members)}>Members</button>
          </TabsTrigger>
          <TabsTrigger value={Page.Policy} asChild>
            <button onClick={() => setPage(Page.Policy)}>Policy</button>
          </TabsTrigger>
          <TabsTrigger value={Page.Settings} asChild>
            <button onClick={() => setPage(Page.Settings)}>Settings</button>
          </TabsTrigger>
          <TabsTrigger value={Page.Support} asChild>
            <button onClick={() => setPage(Page.Support)}>Support</button>
          </TabsTrigger>
          <TabsTrigger value={Page.Subscription} asChild>
            <button onClick={() => setPage(Page.Subscription)}>Subscription</button>
```

links should have `href` use `button` instead

Reply:
also use `.map(...)` to DRY